app/src/androidTest/java/com/example/android/courtcounter/robot/CounterTestRobot.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.android.courtcounter.robot
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.example.android.courtcounter.MainActivity
+import com.example.android.courtcounter.R
+import org.hamcrest.Matchers.allOf
+
+class CounterTestRobot {
+
+    val TEAM_A_SCORE = "0"
```

Comment:
Not a big deal but as your test cases grow you might end up with a lot of constants floating around in different classes. To make things cleaner you can create a TestConstants class (see https://github.com/ozsaat/Fruity/blob/master/app/src/androidTest/java/com/osaat/fruity/utils/TestConstants.kt).